modules/ethereum/src/lib.rs

```diff
@@ -370,6 +442,38 @@ impl<T: Trait> Module<T> {
 	}
 }
 
+impl<T: Trait> frame_support::unsigned::ValidateUnsigned for Module<T> {
+	type Call = Call<T>;
+
+	fn validate_unsigned(call: &Self::Call) -> TransactionValidity {
+		match *call {
+			Self::Call::unsigned_import_header(ref header, ref receipts) => {
+				let accept_result = verification::accept_aura_header_into_pool(
+					&BridgeStorage::<T>::new(),
+					&kovan_aura_config(),
+					&kovan_validators_config(),
+					&pool_configuration(),
+					header,
+					receipts.as_ref(),
+				);
+
+				match accept_result {
+					Ok((requires, provides)) => Ok(ValidTransaction {
+						priority: TransactionPriority::max_value(),
```

Comment:
I'd be more conservative here so that it can still be tuned a bit in case there are multiple transactions in the pool.

I'm drafting an issue to Substrate to suggest that all unsigned transactions should always have a `BasePriority` configurable on the runtime level. That way when you have multiple pallets sending unsigned transactions you can make sure they are correctly ordered in the pool.

So I'd suggest here it should be:
```rust
trait Trait {
 ...
  type UnsignedTransactionBasePriority: Get<TransactionPriority>;
  ...
}
...
ValidTransaction {
   priority: T::UnsignedTransactionBasePriority::get() + height,
   ...
}
```

And then we should set the BasePriority to something like `1 << 20`.

Reply:
See top level comment - let's extract everything about configuration to separate issue.